docs/docs/gatsby-cli.md

```diff
@@ -66,121 +66,121 @@ gatsby new
    (Use a different starter)
 ```
 
-See the [Gatsby starters docs](https://www.gatsbyjs.org/docs/gatsby-starters/) for more details.
+अधिक विवरण के लिए [Gatsby शुरुआत डॉक्स](https://www.gatsbyjs.org/docs/gatsby-starters/) देखें।
 
-### `develop`
+### `विकसित करना`
 
-Once you've installed a Gatsby site, go to the root directory of your project and start the development server:
+एक बार जब आप एक Gatsby साइट इनस्टॉल कर लेते हैं, तो अपनी प्रॉजेक्ट की रूट फोल्डर मे जाएँ और डेवेलपमेंट सर्वर शुरू करें:
 
 `gatsby develop`
 
-#### Options
+#### विकल्प
 
-|     Option      | Description                                     |
+|     विकल्प      | विवरण                                     |
 | :-------------: | ----------------------------------------------- |
-| `-H`, `--host`  | Set host. Defaults to localhost                 |
-| `-p`, `--port`  | Set port. Defaults to 8000                      |
-| `-o`, `--open`  | Open the site in your (default) browser for you |
-| `-S`, `--https` | Use HTTPS                                       |
+ `-H`, `--host`  | होस्ट सेट करें। डिफॉल्ट लोकलहोस्ट             |
+| `-p`, `--port`  | पोर्ट सेट करें। डिफॉल्ट ८००० |                      |
+| `-o`, `--open`  | आपके लिए अपने (डिफ़ॉल्ट) ब्राउज़र में साइट खोलता है |
+| `-S`, `--https` | HTTPS का उपयोग करें                                      |
 
-Follow the [Local HTTPS guide](/docs/local-https/)
-to find out how you can set up an HTTPS development server using Gatsby.
+Gatsby का उपयोग करके आप HTTPS डेवलपमेंट सर्वर कैसे सेट कर सकते हैं, यह जानने के लिए [स्थानीय HTTPS गाइड](/docs/local-https/) का उपयोग करें।
 
-#### Preview changes on other devices
+#### अन्य उपकरणों पर प्रीव्यू परिवर्तन
 
-You can use the Gatsby develop command with the host option to access your dev environment on other devices on the same network, run:
+आप एक ही नेटवर्क पर अन्य उपकरणों पर अपने dev एन्वाइरन्मेंट की जानकारीे के लिए होस्ट विकल्प के साथ gatsby develop कमांड का उपयोग कर सकते हैं, चलाएं:
```

Comment:
`जानकारीे` => `जानकारी`

Reply:
you missed this

Reply:
ops. you missed this again

Reply:
Line no: 90 => Please tell me what i missed. here is the screenshot.
![image](https://user-images.githubusercontent.com/27020381/68739629-47f90680-060f-11ea-83d3-2b1a21ebefb1.png)

Reply:
Check the jaankari spelling in this snapshot

<img width="618" alt="Screen Shot 2019-11-13 at 12 45 12 PM" src="https://user-images.githubusercontent.com/4654382/68741387-7ed11b80-0613-11ea-951f-67e392cf4e79.png">

In my laptop it shows like this, can you try editing it and use the one I have added in comment

Reply:
copy-pasted. Please check now.

Reply:
Looks good now. Thanks